SparkPost/2web2ui
# FE-664: move mock API responses out of __integration__

### What Changed
 - Moved the 30-odd `axios` mock API response files out of `__integration__`. This is prep for the rest of FE-664 which will add functional tests for the billing flows.

### How To Test
 - A full test run will verify that the mock API responses are still working: `npm test`

This will not be merged on its own. We'll merge the whole of FE-664 together. See the other 2 PRs for details.
